Makefile

```diff
@@ -4,7 +4,7 @@ docs.generate:
 	# Sets version to the latest tag to simulate a production build, where certain commands are hidden.
 	rm -rf docs/docs/reference/cli
 	if [ -f ~/.rill/config.yaml ]; then mv ~/.rill/config.yaml ~/.rill/config.yaml.tmp; fi;
-	go run -ldflags="-X main.Version=$(shell git describe --tags `git rev-list --tags --max-count=1`)" ./cli docs generate docs/docs/reference/cli/
+	go run -ldflags="-X main.Version=$(bash git describe --tags `git rev-list --tags --max-count=1`)" ./cli docs generate docs/docs/reference/cli/
```

Comment:
Hey, this did not work. It has set the version to an empty string, as you see, the generated CLI docs now include hidden commands, such as `rill admin`, which they shouldn't.

Reply:
This is done as part of https://github.com/rilldata/rill-developer/pull/2483